File: UCF HSPT Project/src/y2015/ketchup.java

import java.io.*;
import java.util.*;

public class ketchup {
   
   public static void main(String[] args) {
      
      // Open in the input
      Scanner in = new Scanner(System.in);
      
      // Read in the number of items
      int numItems = in.nextInt();
      
      // Loop for each item
      for (int itemID = 1; itemID <= numItems; itemID++) {
         // Read in the item's name
         String itemName = in.next();
         
         // Output the correct output (putting item in the phrase)
         System.out.printf("Word #%d: How do you %s?%n", itemID, itemName);
      }
   }
}

File: UCF HSPT Documents/2015/Solutions/ketchup.c

#include <stdio.h>
#include <stdlib.h>

int main()
{
   // Open the input file.
   FILE * ifp = stdin;

   // Read number of test cases.
   int t;
   fscanf(ifp, "%d", &t);

   // Iterate over test cases.
   int i;
   for (i = 0; i < t; i++)
   {
      // Read input string.
      char s[50];
      fscanf(ifp, "%s", &s);

      // Output formatted string as per problem.
      printf("Word #%d: How do you %s?\n", i + 1, s);
   }

   // Return.
   return 0;
}
